### ISSUE TITLE
Make sure that the warnings being filtered are still required

### ISSUE THREAD

**Issue by desilinguist** [EducationalTestingService/rsmtool#417]
I see a bunch of `filterwarnings()` calls in the codebase:

```bash
test_utils.py
417:        warnings.filterwarnings('ignore', category=RuntimeWarning)
421:            warnings.filterwarnings('ignore', category=warning_type)
463:        warnings.filterwarnings('ignore', category=RuntimeWarning)
467:            warnings.filterwarnings('ignore', category=warning_type)
507:        warnings.filterwarnings('ignore', category=RuntimeWarning)
511:            warnings.filterwarnings('ignore', category=warning_type)
543:        warnings.filterwarnings('ignore', category=RuntimeWarning)
547:            warnings.filterwarnings('ignore', category=warning_type)
584:        warnings.filterwarnings('ignore', category=RuntimeWarning)
588:            warnings.filterwarnings('ignore', category=warning_type)

__init__.py
47:warnings.filterwarnings('always',

fairness_utils.py
263:    np.warnings.filterwarnings('ignore')

modeler.py
35:    warnings.filterwarnings('ignore', category=FutureWarning)

reader.py
263:            warnings.filterwarnings('ignore', category=pd.io.common.DtypeWarning)
```

We should make sure that these are still needed.

**Comment by aloukina** [comment]
See also https://github.com/EducationalTestingService/rsmtool/issues/331

**Comment by desilinguist** [comment]
Closed as duplicate of #331
